### DISCUSSION TITLE
LiveBot v1.3.1 alpha release

### DISCUSSION THREAD

**Discussion by SebOuellette** [SebOuellette/LiveBot#242] [Q&A]
# LiveBot v1.3.1-alpha
### Bug fixes and back-end updates.

> Please ignore the fact that 1.3.0 didn't work at all, and use 1.3.1 instead, which actually works :)

## General Fixes
* Updates Twemoji again, this time to a maintained version on jsdeliver and npm. 
  > v1.2.0's Twemoji will continue to work, as I will continue to host the script. 

* Fixes the intent image in the "Stuck Loading?" menu
* Updates discord.js to version 14.9.0

## What's Changed
* Update discord.js and electron & prettier code by @Xaliks in https://github.com/SebOuellette/LiveBot/pull/231
* Change CDN URL for Twemoji by @PlavorSeol in https://github.com/SebOuellette/LiveBot/pull/232
* Fix shortcuts regex by @Xaliks in https://github.com/SebOuellette/LiveBot/pull/240

## New Contributors
* @PlavorSeol made their first contribution in https://github.com/SebOuellette/LiveBot/pull/232

**Full Changelog**: https://github.com/SebOuellette/LiveBot/compare/v1.2.0-alpha...v1.3.0-alpha

# [NEW] Signing releases with PGP
From now on, I will be signing LiveBot releases using PGP. You can use my public key which can be found on my website here [https://cdn.honeybeeks.net/raw/pgp/seb_ouellette](https://cdn.honeybeeks.net/raw/pgp/seb_ouellette)

Do NOT trust signatures from any key which says it is from me, if it does not match this key found on honeybeeks.net. 

You may download without verifying the signature, but it is recommended. The advantages of verifying the release include:
* Ensuring file integrity (Did the LiveBot release zip download correctly?)
* Ensuring the file has not been modified by malware (Before extracting)

<hr /><em>This discussion was created from the release <a href='https://github.com/SebOuellette/LiveBot/releases/tag/v1.3.1-alpha'>LiveBot v1.3.1 alpha release</a>.</em>

**Comment by ghost** [comment]
hey i cannot download the files it says "The proedure entry point DiscardVirtualMemory could not be located in the dynamic E:\livebot-win32-x64\LiveBot.exe

**Reply by SebOuellette** [reply]
Try downloading again.

**Comment by Meomeo69420** [comment]
![image](https://github.com/SebOuellette/LiveBot/assets/138655024/4362fe02-1a63-49c4-910c-8f1c692918c3)
i cant open it

**Comment by Meomeo69420** [comment]
reply me on meomeo69420 
in the discord
or Kid named finger#7884

**Comment by Colly001** [comment]
Android when?

**Reply by SebOuellette** [reply]
Step 1. Download termux
Step 2. Download VNC
Step 3. Install Proot-distro
Step 4. Setup vnc server on the proot-distro
Step 5. Livebot on Android


Otherwise, a livebot build for Android is likely never going to happen. That's a lot of work that could be better spent developing features imo.

**Comment by amonguserEr** [comment]
is there gonna be any for mac (unless there is one and im stupid)

**Reply by christianfurr** [reply]
its darwin file

i was able to open it an load my bot just fine but other then that i coulnt do anything

**Comment by ghost** [comment]
Is it just me or does this thing just not work at all

**Comment by SubjectKit** [comment]
Hi! When I open it and input my token, it's stuck on the "Logging in" loading screen, is there a fix to this?

**Comment by NikNikovsky** [comment]
"You may download without verifying the signature, but it is recommended."
I think someone meant to say "but it isn't recommended" since after that the benefits are listed

**Reply by SebOuellette** [reply]
No, "it is recommended" to verify the release. Those benefits are the ones you get if you verify the release, which is why it IS recommended.

**Comment by jikothereal** [comment]
it doesn't show any server, i have all gateway thingies enabled, the owner of the bot (me) is in the server where the bot is, but i can't see any dms or Servers, Please help?
![image](https://github.com/user-attachments/assets/1c933219-9c1a-48e2-84f6-005940850952)

**Comment by FoxlynYT** [comment]
Does this still work?

**Comment by FoxlynYT** [comment]
Is there a new good tutorial for this if it still works?

**Reply by SebOuellette** [reply]
https://youtu.be/c4J8TC0-DNI?si=CdJdm-2A7U51oq0x

Here is a link to our official tutorial. Yes it still works.
